### DISCUSSION TITLE
question about Fine-tuning Presets for SevenNet-Omni

### DISCUSSION THREAD

**Discussion by yerinejeong** [MDIL-SNU/SevenNet#271] [Q&A]
Hello everyone,

I'm impressed by the performance of the latest SevenNet-Omni model.

I would like to fine-tune this model for my own dataset. Could you provide a recommended preset (YAML file) or specific hyperparameters (LR, loss weights, etc.) optimized for SevenNet-omni?

The current presets in the repository seem to be for previous versions, so I'd appreciate any guidance for the Omni architecture.

Thank you!

**Comment by Jaesun0912** [comment]
Hi @yerinejeong,
I'm the one who trained 7net-Omni and also responsible for maintaining SevenNet.
Thank you for your attention on our recent research.
Followings are my opinion to your question.

1) Hyperparameters (except that decide model architecture) for fine-tuning are not neccessarily to be equal to that of pretraining. Fine-tuning database would have different distribution with pretrained set (e.g., distrubution of energy / force / stress), so I recommend to use different hyperparameter accordingly after testing. For learning rate, I usually reduce them from pretraining value to prevent catastrophic forgetting.

2) Nevertheless, I agree that preset that are utilized for training 7net-Omni might be a good starting point for fine-tuning. Unfortunately, preset for training 7net-Omni are not yet opened to the public repository, since some newly defined loss functions (e.g., L2MAE loss and selective regularization loss employed in training 7net-Omni) and training code need to be merged beforehand.

3) Hyperparameter you requested (LR, loss weights) can be found in Methods section of our arXiv paper (https://arxiv.org/pdf/2510.11241). We utilized LR = 0.002, energy loss weight = force loss weight = 1, stress loss weight = 1e-4. However, we utilize L2MAE for force and stress loss, which are not merged yet.

If you have any further questions or any problem occurs while fine-tuning 7net-Omni, I would be happy to answer your questions.

**Reply by yerinejeong** [reply]
Hi @Jaesun0912,

Thanks for the quick and helpful reply! I'll make sure to check the arXiv paper for the baseline numbers and adjust the hyperparameters as you advised.

Thanks again for your support, and I'll let you know if I have any further questions.
